zendesk/zendesk.go

```diff
@@ -43,6 +43,7 @@ type Client interface {
 	ListOrganizationTickets(int64, *ListOptions, ...SideLoad) (*ListResponse, error)
 	ListExternalIDTickets(string, *ListOptions, ...SideLoad) (*ListResponse, error)
 	ListRequestedTickets(int64) ([]Ticket, error)
+	ListTicketAudits(ticketID int64, options *ListOptions) (*ListResponse, error)
```

Comment:
For consistency with the other method definitions, I recommend leaving out the variables `ticketID` and `options`